core/modules/views/js/ajax_view.js

```diff
@@ -57,7 +57,7 @@ Backdrop.views.ajaxView = function(settings) {
   this.settings = settings;
 
   // Add the ajax to exposed forms.
-  this.$exposed_form = $('form#views-exposed-form-'+ settings.view_name.replace(/_/g, '-') + '-' + settings.view_display_id.replace(/_/g, '-'));
+  this.$exposed_form = this.$view.children('.view-filters').children('form');
```

Comment:
I believe this may cause Views that use a block for their exposed filters to break, because the ".view-filters" element would be outside of the view wrapper.